mico-core/src/main/java/io/github/ust/mico/core/model/MicoApplication.java

```diff
@@ -177,6 +182,21 @@
     )})
     private String owner;
 
+    /**
+     * Constructor for a new {@link MicoApplication} object with all required data.
+     *
+     * @param shortName   the short name of the service
+     * @param version     the version of the service
+     * @param name        the name of the service
+     * @param description the description of the service
+     */
+    public MicoApplication(String shortName, String version, String name, String description) {
```

Comment:
Why do you add the constructor? It should be enough to use the Lombok annotation **RequiredArgsConstructor**.

Reply:
If I use `RequiredArgsConstructor` also the property `services` is required. However I will consider it in conjunction with the Spring configuration `spring.jackson.default-property-inclusion=NON_NULL`.

Reply:
I removed the constructor again, because description is not really an important property. It it is omitted, it will be set to an empty string that is totally valid.